vms/shift/templates/shift/edit_hours.html

```diff
@@ -1,6 +1,22 @@
 {% extends "vms/base.html" %}
 
 {% block content %}
+<script type="text/javascript">
```

Comment:
@vubo Why don't we need a timepicker everywhere?

I didn't get your second question. :(